src/Razor/src/Microsoft.AspNetCore.Razor.ProjectEngineHost/RazorProjectInfoHelpers.cs

```diff
@@ -15,60 +19,69 @@
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.Diagnostics;
 using Microsoft.CodeAnalysis.Razor;
-using Microsoft.Extensions.Logging;
 
-namespace Microsoft.AspNetCore.Razor.ExternalAccess.RoslynWorkspace;
+namespace Microsoft.AspNetCore.Razor;
 
-internal static class RazorProjectInfoFactory
+internal static class RazorProjectInfoHelpers
 {
     private static readonly StringComparison s_stringComparison;
 
-    static RazorProjectInfoFactory()
+    static RazorProjectInfoHelpers()
     {
         s_stringComparison = RuntimeInformation.IsOSPlatform(OSPlatform.Linux)
             ? StringComparison.Ordinal
             : StringComparison.OrdinalIgnoreCase;
     }
 
-    public static async Task<RazorProjectInfo?> ConvertAsync(Project project, ILogger? logger, CancellationToken cancellationToken)
```

Comment:
is removing log here intentional?

Reply:
Yes, unfortunately when moving the layering we no longer reference MS.Extensions.Logging

Some of the logging was moved before the call to verify